packages/appstore/src/components/wallet-header/__tests__/wallet-header.spec.tsx

```diff
@@ -0,0 +1,326 @@
+            render(
+                <StoreProvider store={mocked_store}>
+                    <WalletHeader wallet_account={mocked_props} />
+                </StoreProvider>
+            );
+
+            expect(screen.getByRole('button', { name: /Transfer/i })).toBeInTheDocument();
+        });
+
+        it('Arrow button click and switchAccount should be called', async () => {
+            const mocked_props = {
+                ...default_mocked_props,
+                balance: 0,
+                is_selected: false,
+                is_demo: true,
+                loginid: 'CRW1231',
+            };
+
+            render(
+                <StoreProvider store={mockedRootStore}>
+                    <WalletHeader wallet_account={mocked_props} />
+                </StoreProvider>
+            );
+
+            const arrow_btn = screen.getByTestId('dt_arrow');
+            userEvent.click(arrow_btn);
+
+            await waitFor(() => {
+                expect(mockedRootStore.client.switchAccount).toBeCalledTimes(1);
+            });
+        });
+
+        it('Check buttons for demo', () => {
+            const mocked_props = {
+                ...default_mocked_props,
+                balance: 0,
+                currency: 'EUR',
+                is_demo: true,
+                loginid: 'VRW123123',
+            };
+
+            const mocked_store = mockStore({
+                client: {
+                    loginid: 'VRW123123',
+                },
+            });
+
+            render(
+                <StoreProvider store={mocked_store}>
+                    <WalletHeader wallet_account={mocked_props} />
+                </StoreProvider>
+            );
+
+            const btn1 = screen.getByRole('button', { name: /Transfer/i });
+            const btn2 = screen.getByRole('button', { name: /Transactions/i });
+            const btn3 = screen.getByRole('button', { name: /Reset balance/i });
+
+            expect(btn1).toBeInTheDocument();
+            expect(btn2).toBeInTheDocument();
+            expect(btn3).toBeInTheDocument();
+        });
+
+        it('Check buttons for real', () => {
+            const mocked_props = {
+                ...default_mocked_props,
+                balance: 1230,
+                currency: 'EUR',
+                loginid: 'CRW123123',
+            };
+
+            const mocked_store = mockStore({
+                client: {
+                    loginid: 'CRW123123',
+                },
+            });
+
+            render(
+                <StoreProvider store={mocked_store}>
+                    <WalletHeader wallet_account={mocked_props} />
+                </StoreProvider>
+            );
+
+            const btn1 = screen.getByRole('button', { name: /Deposit/i });
+            const btn2 = screen.getByRole('button', { name: /Withdraw/i });
+            const btn3 = screen.getByRole('button', { name: /Transfer/i });
+            const btn4 = screen.getByRole('button', { name: /Transactions/i });
```

Comment:
```suggestion
            const deposit_btn = screen.getByRole('button', { name: /Deposit/i });
            const withdraw_btn = screen.getByRole('button', { name: /Withdraw/i });
            const transfer_btn = screen.getByRole('button', { name: /Transfer/i });
            const transactions_btn = screen.getByRole('button', { name: /Transactions/i });
```
same for the rest of numbered buttons in the file